.wiki/how-to-export-and-import-a-site-with-wp-ultimo-site-exporter.md

```
@@ -1,20 +1,20 @@
-# How to Export and Import a Site with WP Multisite WaaS Site Exporter
+# How to Export and Import a Site with Multisite Ultimate Site Exporter
 
-With WP Multisite WaaS you can export a site from your network or import a site to your multisite or single-site installation.
+With Multisite Ultimate you can export a site from your network or import a site to your multisite or single-site installation.
 
-To start the process, go to our add-ons page and install the WP Multisite WaaS: Site Exporter & Importer add-on.![](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-Lqw9doe4z7gj8zUEL9fr6rLILFhIzxlBcvuGXbQEzvFaaF23mTxT0eUoZF9U2-vriFmtQ0S7GApzZfTHmtX2Y30WT0ZOLaNyJz9NJXgulN-m6lxDLnTj8ZkUbB4pjFsuwg=s0)
+To start the process, go to our add-ons page and install the Multisite Ultimate: Site Exporter & Importer add-on.![](https://wp-ultimo-space.fra1.cdn.digitaloceanspaces.com/hs-Lqw9doe4z7gj8zUEL9fr6rLILFhIzxlBcvuGXbQEzvFaaF23mTxT0eUoZF9U2-vriFmtQ0S7GApzZfTHmtX2Y30WT0ZOLaNyJz9NJXgulN-m6lxDLnTj8ZkUbB4pjFsuwg=s0)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)